automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainWorkspaceSpec.scala

```diff
@@ -511,29 +510,6 @@ class AuthDomainWorkspaceSpec extends FreeSpec /*with ParallelTestExecution*/ wi
               }
             }
           }
-          "when the user is a billing project owner" - {
```

Comment:
removed this test because it's same as `when the user is a billing project owner - can be seen but is not accessible`

Reply:
Sorry for snooping but I'm very interested in this. Have you looked at the history of `AuthDomainWorkspaceSpec` and the original `AuthDomainSpec` (which I figure is where this came from) to see where the duplication came from? I worry that there was once a subtle difference that was accidentally lost.

Reply:
np. I'll look in original and compare two tests.

Reply:
@breilly2 Links to two tests in original file on Jan 26. Two tests look identical, perform same tasks.

https://github.com/broadinstitute/firecloud-ui/blob/df7dc9aa061f0e8db5a0f6123ca21ece0e974176/automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala#L360

https://github.com/broadinstitute/firecloud-ui/blob/df7dc9aa061f0e8db5a0f6123ca21ece0e974176/automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala#L394

Reply:
Thanks for checking. May need to go back further in the history of `AuthDomainSpec`. The first one is in a section described as `"when shared with them"` and the other is in `"when not shared with them"` so the difference was probably lost earlier. I originally wrote these tests so I can assure you that there was a difference at one time.

Reply:
@breilly2 Brand new AuthDomainSpec test added on Aug 1, 2017 (https://github.com/broadinstitute/firecloud-ui/pull/852). The branch did not contain these two tests. Matt Bermis added new tests to this Spec on Aug 25, 2017 (https://github.com/broadinstitute/firecloud-ui/pull/896). AuthDomainSpec contained two identical tests.

https://github.com/broadinstitute/firecloud-ui/blob/b336e80a0fb2ea41aa9def719b7d2e26e922ced2/automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala#L324

https://github.com/broadinstitute/firecloud-ui/blob/b336e80a0fb2ea41aa9def719b7d2e26e922ced2/automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala#L359

Reply:
@MatthewBemis @breilly2 refactored dup test to add test coverage.